src/app/[locale]/(default)/rules/(main)/loading.tsx

```diff
@@ -0,0 +1,9 @@
+import { RuleCardListSkeleton } from '@/components/rules/RuleCardListSkeleton';
+
+export default function RulesSkeleton() {
+  return (
+    <>
+      <RuleCardListSkeleton />
+    </>
```

Comment:
The RuleCardListSkeleton component is quite small, so you may move it directly to `loading.tsx`.

If you would still prefer to keep the component as the same file, change the following:
```suggestion
  return <RuleCardListSkeleton />;
```